spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java

```diff
@@ -385,15 +411,26 @@ public DeltaWriter<InternalRow> createWriter(int partitionId, long taskId) {
               .build();
 
       if (command == DELETE) {
-        return new DeleteOnlyDeltaWriter(table, writerFactory, deleteFileFactory, context);
+        return new DeleteOnlyDeltaWriter(
```

Comment:
Add some tests for Delete (right now there's update tests) so we can exercise that path explicitly. The rewrite position deletes procedure tests also exercise this path.